### PR TITLE
Descriptions fixes

### PR DESCRIPTION
**Pull request by shprotru**
fix for #1168 and #1163

### CODE REVIEW

File: src/fheroes2/heroes/skill.cpp [hunk 1]
```diff
@@ -439,7 +439,13 @@ std::string Skill::Secondary::GetDescription( void ) const
...
     switch ( Skill() ) {
     case PATHFINDING:
-        str = _n( "Reduces the movement penalty for rough terrain by %{count} percent.", "Reduces the movement penalty for rough terrain by %{count} percent.", count );
+        if ( this->Level() >= 3 ) {
+            str = _( GetName() + std::string( " eliminates the movement penalty for rough terrain." ) );
+        }
+        else {
+            str = _n( std::string( std::string( GetName() ) + std::string( " reduces the movement penalty for rough terrain by %{count} percent." ) ).c_str(),
+                      std::string( std::string( GetName() ) + std::string( " reduces the movement penalty for rough terrain by %{count} percent." ) ).c_str(), count );
+        }
```
Comment (ihhub) [inline src/fheroes2/heroes/skill.cpp:448]: Hi @shprotru , could you please explain why do we have different calls: first call contains only 1 string but in else condition we have 2 equal strings?

Comment (ihhub) [reply src/fheroes2/heroes/skill.cpp:448]: For the second case if we really need 2 string we have to use a temporary variable where we store the string and then use it.

Comment (shprotru) [reply src/fheroes2/heroes/skill.cpp]: I scroll code down and see there `WISDOM` skill description forming with `switch` statement, changed my code to this style + as you propose, hope it is better now, check, please

**Comment by ihhub** [comment]
Well done, @shprotru ! Great job!

**Pull request merged**